public/app/features/dashboard/components/SubMenu/DashboardLinksDashboard.tsx

```diff
@@ -79,6 +81,16 @@ export class DashboardLinksDashboard extends PureComponent<Props, State> {
     );
   };
 
+  getDropdownLocationCssClass = (): string => {
+    const [pullLeftCssClass, pullRightCssClass] = ['pull-left', 'pull-right'];
+    const wrapper = this.wrapperRef.current;
+    const list = this.listItemRef.current;
+    if (!wrapper || !list) {
+      return pullRightCssClass;
+    }
+    return wrapper.offsetLeft > list.offsetWidth - wrapper.offsetWidth ? pullRightCssClass : pullLeftCssClass;
```

Comment:
this logic does not make sense to me, should not the viewport width be part of this calculation?